.changeset/tasty-buses-jog.md

```diff
@@ -10,8 +10,70 @@ This is a breaking change.
 
 There are two ways to update:
 
-1. Set `experimental.useLegacyManyRelationNames: true` at the root of your config. This will make Keystone revert to the old relation names. This feature will be removed in a future release.
-2. Or, rename your many relation tables using a migration (see below)
+### Set `db.relationName` on many to many relation
+
+To avoid doing a migration, you can set `db.relationName` to the previous relation name on one side of a many to many relationship.
```

Comment:
```suggestion
Rather than doing a migration, you can set the new field property `db.relationName` on any side of a many to many relationship.
If set to the existing relation name, e.g `Post_tags_Tag_posts`, you will not need to run any migration code.
```